content/en/Cobalt-API/v1/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Cobalt REST API v1 docs"
+linkTitle: "API v1 docs"
+weight: 201
+description: >
+  The documentation for v1 of the public Cobalt API.
+---
+
+Cobalt REST API v1 docs
```

Comment:
@ericcornelissen I think we can make these hidden from the nav as we don't want to point folks here

Reply:
Done in e08c518a651756edd594b85be4e1b3488cb6ecd4